content/13.conclusion.md

```diff
@@ -1,13 +1,10 @@
 ## Conclusion {#conclusion}
 
-Collectively, our manuscript is focused on the promotion of practical tips distilled from cutting-edge insights and evolving professional standards to advance the efficient and optimal application of deep learning within research.
-It is evident that some of our points (see Tips 7, 8, 9, and 10) are intimately linked to safeguarding against key risks: for example, introduction/perpetuation of bias, overinterpretation/misinterpretation of models, poor generalizability, and potential for harm unto others—which can have a mix of ethical, legal, and social implications.
-If leveraged in ethical and responsible ways, deep learning techniques have the potential to add value within a diverse array of research and healthcare contexts, as these techniques have already shown remarkable capacity to meet or exceed the performance of human effort or older algorithms across fields and subdisciplines.
-Beyond merely achieving good predictive performance in certain tasks, deep learning has the potential to uncover high-impact biological and clinical insights, fundamentally driving research discoveries and delivery of new products to market.
-Yet, to realize its full potential, deep learning must be approached by all with genuine thoughtfulness, caution, and responsibility.
+Collectively, our manuscript is focused on the promotion of practical tips, cutting-edge insights, and evolving professional standards that can help to advance the efficient and optimal application of deep learning across a wide range of scientific research environments.
+Specifically, we have tried to increase the accessibility of deep learning techniques within the life sciences so that these cutting edge techniques can generate innovative biomedical insights, tools, technologies, services, and products.
```

Comment:
```suggestion
Specifically, we have tried to increase the accessibility of deep learning techniques within the life sciences so that these these methods can generate innovative biomedical insights, tools, technologies, services, and products.
```